Quarantine BlazorWasmHostedTemplate_AzureActiveDirectoryTemplate_*

<!--
Note this issue template is specifically for failing tests within the dotnet/aspnetcore repo.
-->

## Failing Test(s)

- Templates.Blazor.Test.BlazorWasmTemplateAuthTest.BlazorWasmHostedTemplate_AzureActiveDirectoryTemplate_IndividualB2C_Works
- Templates.Blazor.Test.BlazorWasmTemplateAuthTest.BlazorWasmHostedTemplate_AzureActiveDirectoryTemplate_IndividualB2C_NoHttps_Works
- Templates.Blazor.Test.BlazorWasmTemplateAuthTest.BlazorWasmHostedTemplate_AzureActiveDirectoryTemplate_SingleOrg_Works
- Templates.Blazor.Test.BlazorWasmTemplateAuthTest.BlazorWasmHostedTemplate_AzureActiveDirectoryTemplate_SingleOrg_NoHttps_Works
- Templates.Blazor.Test.BlazorWasmTemplateAuthTest.BlazorWasmHostedTemplate_AzureActiveDirectoryTemplate_SingleOrg_ProgramMain_Works
- Templates.Blazor.Test.BlazorWasmTemplateAuthTest.BlazorWasmHostedTemplate_AzureActiveDirectoryTemplate_SingleOrg_NoHttps_ProgramMain_Works

## Error Message

<!--
Provide the error message associated with the test failure, if applicable.
-->

```text
error MSB4018: The "GenerateBlazorBootExtensionJson" task failed unexpectedly.
error MSB4018: System.IO.IOException: The process cannot access the file
```

## Stacktrace

<details>
<!--
Provide the stack trace associated with the test failure, if applicable.
-->

```text
Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets(160,5): error MSB4018: System.IO.IOException: The process cannot access the file '/datadisks/disk1/work/B203097C/w/A4B70922/e/Templates/BaseFolder/AspNet.chhgnoopnlfy/Client/obj/Release/net8.0/blazor.publish.boot-extension.json' because it is being used by another process. [/datadisks/disk1/work/B203097C/w/A4B70922/e/Templates/BaseFolder/AspNet.chhgnoopnlfy/Client/AspNet.chhgnoopnlfy.Client.csproj]
Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets(160,5): error MSB4018:    at Microsoft.Win32.SafeHandles.SafeFileHandle.Init(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Int64& fileLength, UnixFileMode& filePermissions) [/datadisks/disk1/work/B203097C/w/A4B70922/e/Templates/BaseFolder/AspNet.chhgnoopnlfy/Client/AspNet.chhgnoopnlfy.Client.csproj]
Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets(160,5): error MSB4018:    at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, UnixFileMode openPermissions, Int64& fileLength, UnixFileMode& filePermissions, Func`4 createOpenException) [/datadisks/disk1/work/B203097C/w/A4B70922/e/Templates/BaseFolder/AspNet.chhgnoopnlfy/Client/AspNet.chhgnoopnlfy.Client.csproj]
Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets(160,5): error MSB4018:    at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode) [/datadisks/disk1/work/B203097C/w/A4B70922/e/Templates/BaseFolder/AspNet.chhgnoopnlfy/Client/AspNet.chhgnoopnlfy.Client.csproj]
Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets(160,5): error MSB4018:    at System.IO.File.Create(String path) [/datadisks/disk1/work/B203097C/w/A4B70922/e/Templates/BaseFolder/AspNet.chhgnoopnlfy/Client/AspNet.chhgnoopnlfy.Client.csproj]
Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets(160,5): error MSB4018:    at Microsoft.NET.Sdk.BlazorWebAssembly.GenerateBlazorBootExtensionJson.Execute() [/datadisks/disk1/work/B203097C/w/A4B70922/e/Templates/BaseFolder/AspNet.chhgnoopnlfy/Client/AspNet.chhgnoopnlfy.Client.csproj]
Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets(160,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute() [/datadisks/disk1/work/B203097C/w/A4B70922/e/Templates/BaseFolder/AspNet.chhgnoopnlfy/Client/AspNet.chhgnoopnlfy.Client.csproj]
Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets(160,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask) [/datadisks/disk1/work/B203097C/w/A4B70922/e/Templates/BaseFolder/AspNet.chhgnoopnlfy/Client/AspNet.chhgnoopnlfy.Client.csproj]
```
</details>


## Logs

https://helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-aspnetcore-refs-pull-47540-merge-f27eaa5066f44a74be/Templates.Blazor.WebAssembly.Auth.Tests--net8.0/4/console.f2e43e1a.log?helixlogtype=result

## Build

<!--
Provide a link to the build where the test failure occurred.
-->

https://dev.azure.com/dnceng-public/public/_build/results?buildId=254119&view=results

--- a/src/Middleware/ResponseCaching/test/ResponseCachingMiddlewareTests.cs
+++ b/src/Middleware/ResponseCaching/test/ResponseCachingMiddlewareTests.cs
@@ -441,7 +441,6 @@ public class ResponseCachingMiddlewareTests
     }
 
     [Fact]
-    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/48373")]
     public void FinalizeCacheHeadersAsync_ResponseValidity_UseExpiryIfAvailable()
     {
         var timeProvider = new MockTimeProvider();

--- a/src/ProjectTemplates/test/Templates.Blazor.WebAssembly.Auth.Tests/BlazorWasmTemplateAuthTest.cs
+++ b/src/ProjectTemplates/test/Templates.Blazor.WebAssembly.Auth.Tests/BlazorWasmTemplateAuthTest.cs
@@ -95,37 +95,31 @@ public class BlazorWasmTemplateAuthTest : BlazorTemplateTest
 
     [ConditionalTheory]
     [MemberData(nameof(TemplateDataIndividualB2C))]
-    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/47933")]
     public Task BlazorWasmHostedTemplate_AzureActiveDirectoryTemplate_IndividualB2C_Works(TemplateInstance instance)
         => CreateBuildPublishAsync(auth: instance.Auth, args: instance.Arguments, targetFramework: "netstandard2.1");
 
     [ConditionalTheory]
     [MemberData(nameof(TemplateDataIndividualB2C))]
-    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/47933")]
     public Task BlazorWasmHostedTemplate_AzureActiveDirectoryTemplate_IndividualB2C_NoHttps_Works(TemplateInstance instance)
         => CreateBuildPublishAsync(auth: instance.Auth, args: instance.Arguments.Union(new[] { ArgConstants.NoHttps }).ToArray(), targetFramework: "netstandard2.1");
 
     [ConditionalTheory]
     [MemberData(nameof(TemplateDataSingleOrg))]
-    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/47933")]
     public Task BlazorWasmHostedTemplate_AzureActiveDirectoryTemplate_SingleOrg_Works(TemplateInstance instance)
         => CreateBuildPublishAsync(auth: instance.Auth, args: instance.Arguments, targetFramework: "netstandard2.1");
 
     [ConditionalTheory]
     [MemberData(nameof(TemplateDataSingleOrg))]
-    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/47933")]
     public Task BlazorWasmHostedTemplate_AzureActiveDirectoryTemplate_SingleOrg_NoHttps_Works(TemplateInstance instance)
         => CreateBuildPublishAsync(auth: instance.Auth, args: instance.Arguments.Union(new[] { ArgConstants.NoHttps }).ToArray(), targetFramework: "netstandard2.1");
 
     [ConditionalTheory]
     [MemberData(nameof(TemplateDataSingleOrgProgramMain))]
-    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/47933")]
     public Task BlazorWasmHostedTemplate_AzureActiveDirectoryTemplate_SingleOrg_ProgramMain_Works(TemplateInstance instance)
         => CreateBuildPublishAsync(auth: instance.Auth, args: instance.Arguments, targetFramework: "netstandard2.1");
 
     [ConditionalTheory]
     [MemberData(nameof(TemplateDataSingleOrgProgramMain))]
-    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/47933")]
     public Task BlazorWasmHostedTemplate_AzureActiveDirectoryTemplate_SingleOrg_NoHttps_ProgramMain_Works(TemplateInstance instance)
         => CreateBuildPublishAsync(auth: instance.Auth, args: instance.Arguments.Union(new[] { ArgConstants.NoHttps }).ToArray(), targetFramework: "netstandard2.1");
 }

--- a/src/Servers/Kestrel/Transport.Quic/test/QuicConnectionListenerTests.cs
+++ b/src/Servers/Kestrel/Transport.Quic/test/QuicConnectionListenerTests.cs
@@ -371,7 +371,6 @@ public class QuicConnectionListenerTests : TestApplicationErrorLoggerLoggedTest
 
     [ConditionalFact]
     [MsQuicSupported]
-    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/48678")]
     public async Task AcceptAsync_NoCertificateCallback_RemovedFromPendingConnections()
     {
         // Arrange

--- a/src/Servers/Kestrel/test/InMemory.FunctionalTests/Http2/Http2WebSocketTests.cs
+++ b/src/Servers/Kestrel/test/InMemory.FunctionalTests/Http2/Http2WebSocketTests.cs
@@ -152,7 +152,6 @@ public class Http2WebSocketTests : Http2TestBase
     }
 
     [Fact]
-    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/42133")]
     public async Task HEADERS_Received_SecondRequest_Accepted()
     {
         var appDelegateTcs = new TaskCompletionSource();

